.storybook/manager-head.html

```diff
@@ -0,0 +1,9 @@
+<!-- Favicons -->
+<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+<link rel="manifest" href="/site.webmanifest">
+<link rel="shortcut icon" href="/favicon.ico">
+<meta name="msapplication-TileColor" content="#da532c">
```

Comment:
`TileColor`is set in `browserconfig.xml`. Isn't that enough?

Reply:
These settings are the default, when you use the favicon generator, which is widely being used. I guess they know what they are doing, but I never checked that.

https://realfavicongenerator.net/